content/adopting-an-ethical-license.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Adopting an Ethical License"
+heading = "Adopting an Ethical License"
++++
+
+If you are starting a new project, you _may_ [review existing ethical licenses](./licenses/) and choose the one that most directly matches your goals.
+
+For maintainers considering adopting an ethical license on an existing project, be aware that this requires significant work. Each license is different, which makes a general guide difficult to write. Bear in mind that this is not legal advice, and we are not lawyers.
```

Comment:
_may_ require. I personally don't think the MIT sublicense strategy is all that onerous, you just have to dot your i's and cross your t's. But GPL-licensed projects would yes obviously require a lot of work.

Reply:
Even with MIT, it is a significant effort to relicense. I think it's much easier for new contributions to fall under an ethical license than it is to relicense past contributions, and agree with your point that clearly disambiguating the two pathways is a useful structural change.